src/pages/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { signIn, signOut, useSession } from 'next-auth/react';
 
-const Home: React.FC = () => {
```

Comment:
Undo this change since it has nothing to do with this branch and will cause conflicts with other branches